File: src/SIMPLEPROGRAMS/EscapeOperators/EscapeOperators.java
package SIMPLEPROGRAMS.EscapeOperators;

public class EscapeOperators{

    EscapeOperators(){

        System.out.println("\r - CARRIAGE RETURN      \b - BACK SPACE        \t - TAB        \065 - OCTAL       \u0397 - UNICODE VALUE 16 BIT       \u00001000 - UNICODE VALUE 32 BIT        \n - NEW LINE        \f - FORM FEED        \" - DOUBLE QUOTE        \' - SINGLE QUOTE        \\ - BACK SLASH        NULL - \0 ");
    }       
    public static void main(String []args){

        /*
        
(NOT FOR JAVA) \a - ALERT
        \b - BACK SPACE
        \t - TAB
(NOT FOR JAVA) \v - VERTICAL TAB
        \n - NEW LINE
        \f - FORM FEED
        \r - CARRIAGE RETURN
        \" - DOUBLE QUOTE
        \' - SINGLE QUOTE
(NOT FOR JAVA) \? - PRINT QUIZ MARK
        \\ - BACK SLASH
        \0 - NULL
        \ooo - OCTAL
(NOT FOR JAVA) \xhh - HEX DECIMAL
        \Uxxxx - UNICODE VALUE IN 16 BIT
        \Uxxxxxxxx - UNICODE VALUE IN 32 BIT

    */

        EscapeOperators key = new EscapeOperators();    
        
    }
}

File: src/SIMPLEPROGRAMS/EscapeOperators/EscapeOperators.php
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Escape Operators</title>
    <link rel="stylesheet" type="text/css" href="Escape Operators(C).css" />
</head>
<body>
<p id="Head">Escape Operators</p><hr />   

<?php
/*

(NOT FOR PHP)        \a - ALERT
(NOT FOR PHP)        \b - BACK SPACE
        \t - TAB
        \v - VERTICAL TAB
(NOT FOR PHP)        \n - NEW LINE
        \f - FORM FEED
        \r - CARRIAGE RETURN
        \" - DOUBLE QUOTE
(NOT FOR PHP)        \' - SINGLE QUOTE
(NOT FOR PHP) \? - PRINT QUIZ MARK
       \\ - BACK SLASH
        \0 - NULL
        \ooo - OCTAL
        \xhh - HEX DECIMAL
(NOT FOR PHP)        \uxxxx - UNICODE VALUE IN 16 BIT
(NOT FOR PHP)        \uxxxxxxxx - UNICODE VALUE IN 32 BIT

*/

print "\r - CARRIAGE RETURN        \t - TAB     \v - VERTICAL TAB        \f - FORM FEED        \" - DOUBLE QUOTE        \\ - BACK SLASH        \065 - OCTAL        \x24 - HEX DECIMAL     NULL - \0 ";

$xml = simplexml_load_file("Escape Operators(C).xml") or die("unable to load");
echo "<p id='xmlhead'><hr /><br /><br /><br />XML</p><hr />";
echo $xml->INDEX."<hr />";

?>
</body>
</html>